pkg/codegen/templates/imports.tmpl

```diff
@@ -26,7 +27,8 @@ import (
 	"github.com/getkin/kin-openapi/openapi3"
 	"github.com/go-chi/chi/v5"
 	"github.com/labstack/echo/v4"
-	"github.com/gin-gonic/gin"
+	"github.com/gin-gonic/gin"{{ if $globalOpts.Generate.GorillaServer }}
+	"github.com/gorilla/mux"{{ end }}
```

Comment:
were these changes required for this change to work?

Reply:
The code formatting removes unused imports, so we don't usually add them conditionally.

Reply:
Did i understand correctly, what i just need to add `github.com/gorilla/mux` import and remove condition?

Reply:
Yes exactly, just removing the conditional should work.

That being said, I'm not sure if these changes were needed for this PR to work?

Reply:
I can move it to another PR) 
As i said in Issue, without this fix (adding import), users must add additional import to config file.

Reply:
I made a change: remove global var and remove conditional import.